grpc/eg/src/main/protobuf/eg.proto

```diff
@@ -15,7 +15,7 @@ enum Enumeration {
 
 message Message {
   enum Enumeration {
-    ZERO = 0;
+    zero = 0;
```

Comment:
What do you mean? This is here to break previous tests, which are fixed by adding backticks.

CLA signed.